pkg/execute/factory.go

```diff
@@ -7,19 +7,18 @@ import (
 	"github.com/slack-go/slack"
 
 	"github.com/kubeshop/botkube/internal/audit"
+	config2 "github.com/kubeshop/botkube/internal/command"
```

Comment:
```suggestion
	guard "github.com/kubeshop/botkube/internal/command"
```

let's rename that to make it more readable